bonus.mobile-app/CrmApp/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @format
+ */
+
+import App from './App';
```

Comment:
you are not using this component, so you can delete the import and the original file :)